### PR TITLE
Fix starting /camera/get_settings Service

### PR DESCRIPTION
**Pull request by mdhorn**
Fixes Issue: #66 

Changes proposed in this pull request:
- Properly enable /camera/get_settings service
- Added a new GetSettingsService Test

### CODE REVIEW

File: realsense_camera/test/camera_core.cpp [hunk 1]
```diff
@@ -628,6 +634,7 @@ int main(int argc, char **argv) try
   ros::init(argc, argv, "utest");
   fillConfigMap(argc, argv);
 
+
   ROS_INFO_STREAM("RealSense Camera - Starting Tests...");
```
Comment (rjingar) [inline realsense_camera/test/camera_core.cpp]: Minor thing: please remove this extra line 637

Comment (rjingar) [inline realsense_camera/test/camera_core.cpp]: Everything else looks good. Approved.

Comment (mdhorn) [inline realsense_camera/test/camera_core.cpp]: Removed extra white space and resubmitted.

Comment (rjingar) [inline realsense_camera/test/camera_core.cpp]: Approved.

**Comment by kevincwells** [comment]
This looks pretty straight forward.  Approved. :+1:

**Pull request merged**